### PR TITLE
Add PRIMA space

### PR DESCRIPTION
**Pull request by rootdraws**
**Add space for Prima**  
  
Etherscan: https://etherscan.io/token/0xc0B126a67AA9c2995472E75BC66C2F69f955fe16  
Twitter: https://twitter.com/rootdraws
Patreon: https://patreon.com/rootdraws
Kickstarter: https://www.kickstarter.com/profile/rootdraws/created
HIVE: https://peakd.com/@rootdraws

### CODE REVIEW

File: spaces/prima/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.name {
```
Comment (e-asia) [inline spaces/prima/skin.scss:1]: The css selector should similar to the space key.
```suggestion
.prima {
```

File: spaces/prima/skin.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/prima/skin.scss:10]: To use the default theme you will want to delete skin.scss from your space folder.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/prima/skin.scss (hunk 2 above)
Comment (rootdraws) [reply spaces/prima/skin.scss:10]: Hey, can you help me understand how to delete skinn.scss from my space folder?

Comment (e-asia) [reply spaces/prima/skin.scss:10]: You delete skin.scss manually, `git status`, you should see skin css deleted `git add skin.scss`, `git commit -m "delete default skin"` and `git push`. It should work for me :)

**Comment by rootdraws** [comment]
Thanks I'll reach out to you in a week or so, I'm waiting for colony to
update their project, because my ability to mint coins is tied up with
their service, and they've shut it down for a moment. I could do this
elsewhere, but I want to try and use some synergy.

Thank you for the response, talk to you in a bit.

**Comment by e-asia** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**